### ISSUE TITLE
iPad App crashes all the time

### ISSUE THREAD

**Issue by gijoe911** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1434]
As video shows.. the ipad app crashes all the time. we experience this multiples a day and it's getting out of hand. please issue a fix for this. Thanks

https://user-images.githubusercontent.com/25240657/104152431-7d1f0580-541a-11eb-9cc5-b3a7544ab744.mp4

**Comment by Repairdesk-QA** [comment]
Hey @gijoe911, 
Amy here Sure thing! Let me take a look to see if I'm getting the same issue as you and I'll circle back to you.
Thanks & Regards,

**Comment by Repairdesk-QA** [comment]
Hey @gijoe911,
I have checked your concern but unfortunately, I am unable to reproduce your issue. It's working fine for me.
Can you please Logout and Re-login again? It will be working fine for you. Kindly also see the attached video as well.
 https://drive.google.com/file/d/171iaYgQthJVe5XnOoW-8mMowfhzDjkPX/view?usp=sharing
If you have any questions or if there is anything else with which I may assist, please do not hesitate to reach out.
Thanks & Regards,
M Adnan

**Comment by gijoe911** [comment]
yes in fact i've just experienced this same issue 5 minutes ago. i think another video wouldn't make a difference. we experience this issue multiple times everyday across all our stores so please test extensively before coming to a conclusion. Appreciate it!

**Comment by gijoe911** [comment]
HI @Repairdesk-QA this issue still haunts us everyday. can u please look into it?

**Comment by Repairdesk-QA** [comment]
@Repairdesk-QA FYI
